hw-e2e/patient-master.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { test } from "@playwright/test";
+import { CreatePatient } from "../locators/create-patient";
+import { PatientMaster } from "../locators/patient-master";
+import { SideMenu } from "../locators/side-menu";
+import { assert } from "console";
+import { TIMEOUT } from "dns";
+
+test("Test to verify patient Master page is displyed", async ({
+  page,
+  baseURL,
+}) => {
+  await page.goto(`${baseURL}`);
+  const patientMaster = new PatientMaster(page);
+  await patientMaster.heading.isVisible();
+});
+
+test("Test to verify search patient in patient master ", async ({
+  page,
+  baseURL,
+}) => {
+  await page.goto(`${baseURL}`);
+  const patientMaster = new PatientMaster(page);
+  await patientMaster.heading.isVisible();
+  await page.waitForTimeout(1000);
+  await patientMaster.search_enter("james");
+  await patientMaster.verify_search_result();
+});
+
+test("Test to verify search invalid patiend", async ({ page, baseURL }) => {
+  await page.goto(`${baseURL}`);
+  const patientMaster = new PatientMaster(page);
+  await patientMaster.heading.isVisible();
+  await page.waitForTimeout(1000);
+  await patientMaster.search_enter("ASDASDASDasd");
+  await patientMaster.verify_no_data_found();
+});
+
+test("Test to validate criterias", async ({ page, baseURL }) => {
```

Comment:
can we mention that we are validating list of criteria or dropdown values of criteria?

Reply:
added